### ISSUE TITLE
createChannel doesn't respond with the id as per documentation

### ISSUE THREAD

**Issue by eicca** [mainflux/mainflux-core#11]
Hi, I'm just following the getting started docs.  The docs mention that post to the `/channels` should respond with a json: http://mainflux.io/getting-started/#provisioning-channels

However, only the header is being set: https://github.com/mainflux/mainflux-core/blob/master/api/channels.go#L85

**Comment by drasko** [comment]
@eicca yes, you're right - there has been undocumented change in API, pushed by @mijicd recently. He moved response to "Location" header, and I do agree that this is more appropriate place for created resource. I think the same will be done for Device creation.

One thing that can help i to see how the CLI handles it: https://github.com/mainflux/mainflux-cli/blob/master/cmd/channels.go#L32

Point is that Mainflux project is still young, not stabilized yet and written documentation can not keep-up always with changes in the code. So for now I have no better advice than "Use the source, Luke" ;).

We are trying to document all as I know how this frustrating can be, but it is difficult to cover all changes fast due to the complexity and size of the project. All the Mkdocs documentation is kept here: https://github.com/mainflux/mainflux-doc/tree/master/mkdocs and we do accept PRs. 

For urgent questions/explications most of us are hanging on project's IRC: https://gitter.im/Mainflux/mainflux, so do not hesitate to shout over there if you get stuck ;)

**Comment by eicca** [comment]
Cool, thank you very much for the explanation!
